app/js/views/services/light.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'components/react';
+
+import NavigationMenu from 'js/views/navigation-menu';
+import TagList from 'js/views/tag-list';
+
+export default class LightService extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: props.service.tags,
+      tags: []
+    };
+
+    this.foxbox = props.foxbox;
+    this.service = props.service;
+
+    this.onServiceStateChanged = this.onServiceStateChanged.bind(this);
+  }
+
+  componentDidMount() {
+    this.populateTags();
+
+    this.foxbox.addEventListener(
+      'service-state-change', this.onServiceStateChanged
+    );
+  }
+
+  componentWillUnmount() {
+    this.foxbox.removeEventListener(
+      'service-state-change', this.onServiceStateChanged
+    );
+  }
+
+  onServiceStateChanged(service) {
+    if (service.id !== this.service.id) {
+      return;
+    }
+
+    this.service = service;
+    this.setState({ data: service.tags });
+  }
+
+  populateTags() {
+    this.foxbox.getTags()
+      .then(tags => {
+        tags.forEach(tag => {
+          tag.data.checked = !!(this.state.data &&
+            this.state.data.includes(tag.id));
+        });
+
+        this.setState({ tags });
+      });
+  }
+
+  handleAddTag() {
+    let name = prompt('Enter new tag name');
+
+    if (!name || !name.trim()) {
+      return;
+    }
+
+    name = name.trim();
+    this.foxbox.setTag({ name })
+      .then(() => {
+        this.populateTags(); // Needed to get the newly added tag ID.
+      });
+  }
+
+  render() {
+    return (
+      <div className="app-view">
```

Comment:
__issue:__ Ah, sorry I'll need to account for new BaseView, so your render should just render body (probably with ```app-view__fill-body``` class)